### ISSUE TITLE
iOS orientation issue using Pager + HtmlView

### ISSUE THREAD

**Issue by felixkrautschuk** [triniwiz/nativescript-pager#170]
### If the demo apps cannot help and there is no issue for your problem, tell us about it
Please, ensure your title is less than 63 characters long and starts with a capital
letter.

When using an HtmlView as the content of a page within the pager component, changing the orientation of the device is causing some issue on iOS.

When the user opens the first page within pager and then changes the orientation of the device, everything works as expected.
But if the user goes forward and then changes the orientation, the HtmlView suddenly "disappears". And the user has no chance to bring it back to view.

See the following GIF for more details (the Page rbackgroundColor is green, the HtmlView has yellow backgroudColor):

![ios_pager_htmlview_orientation](https://user-images.githubusercontent.com/6443021/77854670-e2876a00-71eb-11ea-8bfb-8c9f300ba1a6.gif)

### Which platform(s) does your issue occur on?
- iOS
- tested on iOS 12.4 and 13.3 / 13.4
- Simulator and device

### Please, provide the following version numbers that your issue occurs with:

- CLI: 6.5.0
- Cross-platform modules: 6.5.0
- Runtime(s): tns-ios 6.5.0
- Plugin(s): nativescript-pager@12.0.0-alpha.8

### Please, tell us how to recreate the issue in as much detail as possible. 
Just do the same steps as shown in the GIF.

### Is there any code involved? 
[ns-pager-htmlview-orientation.zip](https://github.com/triniwiz/nativescript-pager/files/4399279/ns-pager-htmlview-orientation.zip)

### Additional info
When using a Label instead of HtmlView, then everything works as expected.
But using an HtmlView without a Pager component works as expected as well, so this issue seems to be caused by the Pager in some way.

**Comment by jb-jonathan** [comment]
Hello Felix, did you ever find a solution for this issue? I'm running into it as well.

**Comment by triniwiz** [comment]
I think I may have a fix for this 


Sent from Yahoo Mail for iPhone


On Wednesday, October 21, 2020, 5:04 PM, jb-jonathan <notifications@github.com> wrote:




Hello Felix, did you ever find a solution for this issue? I'm running into it as well.

**Comment by felixkrautschuk** [comment]
@jb-jonathan nope, I have not found any solution for that, but I am glad to hear that @triniwiz might have a fix for that.
